Arrowgene.Ddon.Shared/AssetReader/QuestAssetDeserializer.cs

```diff
@@ -92,6 +93,13 @@ private bool ParseQuest(QuestAssetData assetData, JsonElement jQuest)
                 assetData.NewsImageId = jNewsImage.GetUInt32();
             }
 
+            // For the purpose of setting up alternate quests.
+            assetData.VariantId = null;
```

Comment:
Should we use `0` instead of `null` to mean no variant? I find it slightly strange to make a `uint` nullable.

Reply:
Changed :) was left over when the data type was different, my bad